packages/design-picker/src/components/style-variation-badges/utils.ts

```diff
@@ -104,13 +104,16 @@ function findColorBestAnalogous( hexCodes: string[], baseHex: string ) {
 
 export function getStylesColorFromVariation(
 	variation: StyleVariation
-): StyleVariationStylesColor {
+): StyleVariationStylesColor | null {
 	const palette = getColors( variation );
 	const colorBase = getColorBaseFromColors( palette );
 	const colorList = palette.map( ( item ) => item.color );
 
-	return {
-		background: colorBase,
-		text: findColorBestAnalogous( colorList, colorBase ),
-	};
+	try {
+		return { background: colorBase, text: findColorBestAnalogous( colorList, colorBase ) };
+	} catch ( e ) {
+		// eslint-disable-next-line no-console
+		console.error( e, variation );
```

Comment:
Just a minor thing, but I think generally, we should start using `debugFactory` and `debug`, though we don't yet use these in themes code iirc. 

See p1701445784913089/1701434252.464999-slack-C048CUFRGFQ for some discussion.

Reply:
Good point @dsas! Created a PR for this https://github.com/Automattic/wp-calypso/pull/85918